### DISCUSSION TITLE
Ability to use two auth methods at the same time?

### DISCUSSION THREAD

**Discussion by mattoni** [scalar/scalar#1439] [General]
![image](https://github.com/scalar/scalar/assets/5110855/17c5df5e-3e0f-4046-bc6e-5f57d367bd92)

I have an API that utilizes two required authorization methods at the same time (bearer token and a special header). It would be nice if this allowed me to use both at the same time without having to manually provide the custom header for each request.

**Comment by marclave** [comment]
@mattoni yes we agree, lets fix this :)

**Reply by hanspagel** [reply]
Just for reference, we’ve created an issue here: https://github.com/scalar/scalar/issues/1848
